### PR TITLE
HV Threading Crash Fix

### PR DESCRIPTION
**Pull request by BenLand100**
This removes the undo manager from a few high voltage related methods that are called by the init or monitoring threads. The undo manager introduced a race condition leading to a rare crash observed in #77. Also includes a minor change - hv related error messages are now red in the Orca log.

Pull request from Benjamin Land

### CODE REVIEW

File: Source/Objects/Custom Hardware/SNO+/XL3/ORXL3Model.m [hunk 1]
```diff
@@ -2317,7 +2313,7 @@ - (void) writeXl3Mode
         args->dataAvailMask = swapLong(args->dataAvailMask);
     }
 
-    //[self setXl3ModeRunning:YES];
+    [self setXl3ModeRunning:YES];
```
Comment (BenLand100) [inline Source/Objects/Custom Hardware/SNO+/XL3/ORXL3Model.m]: @tlatorre-uchicago these lines are commented out in unstable but not in master, which is probably why my last PR had them commented out. I'm not sure how that happened, but I believe we decided uncommented was correct?

**Comment by BenLand100** [comment]
@tlatorre-uchicago this also now posts notifications to main thread.

**Comment by BenLand100** [comment]
@tlatorre-uchicago besides removing the undo manager calls that resulted in #77, it's conceivable that wrapping all the ORXL3ModelHvStatusChanged notifications in main thread dispatchers might also fix #68. We'd probably need the whole detector to check either because if I recall neither was seen on the teststand, but these should only be cosmetic changes as far as hardware settings are concerned.

**Comment by tlatorre-uchicago** [comment]
Thanks! merged.

**Pull request closed without merging**